### ISSUE TITLE
Caption is rendered below table 

### ISSUE THREAD

**Issue by VTrngNghia** [react-bootstrap-table/react-bootstrap-table2#1185]
**Describe the bug**
A clear and concise description of what the bug is. In addition, please search issues before you open a report to make sure there's no any duplicated report

**To Reproduce**
From the story book, the attribute `caption=` is supposed to render a caption on top of the table
`<BootstrapTable keyField="id" data={ products } caption="Plain text header" columns={ columns } />`

When I tried it, it renders the caption below the table.

**Screenshots**
![image](https://user-images.githubusercontent.com/26227075/69923788-02c34880-14da-11ea-91a8-0beb234f5c57.png)

**codesandbox**
I dont know how to use sandbox, here's my code in React

```
    const tbl_classes = isLoading
      ? <Spinner/>
      : <CardBody>
        <BootstrapTable
          striped hover condensed bordered={false}
          keyField="id" data={classes} columns={columns}
          caption="Caption in plain text"
          noDataIndication="Table is Empty"
        />
      </CardBody>;

```

**Comment by Ymbere** [comment]
Hi @VTrngNghia, your code is right the issue appears to be with the bootstrap4 CSS.
If it helps, for now, you can change your bootstrap CSS to the version 3.x.x

### Here is a working example using bootstrap3
[![Edit bootstrap3-table-caption-test](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/react-new?fontsize=14&hidenavigation=1&theme=dark)

I will try to solve that issue as soon as I can.

**Comment by AllenFang** [comment]
Fixed in `react-bootstrap-table-next@3.3.3`, https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2019/12/07/version-bump.html
